symtabAPI/src/Object-elf.C

```diff
@@ -4184,3 +4257,210 @@ bool ObjectELF::getRegValueAtFrame(Dyninst::Address pc, Dyninst::MachRegister re
    return dwarf->frameParser()->getRegValueAtFrame(pc, reg, reg_result, reader, frame_error);
 
 }
+
+bool ObjectELF::parse_attrs(const char *attr_string,
```

Comment:
Oh, but for want of a [`string_view`](https://www.boost.org/doc/libs/latest/libs/utility/doc/html/utility/utilities/string_view.html)...

That would allow using iterators instead of keeping an array index around. You don't need to update this now, but it's definitely something to consider for the future.

Reply:
Yes, I do have some test attributes to verify the correctness.
@hainest Where should I add the unit test?